src/arrayMethodJoin.js

```diff
@@ -4,9 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joined = '';
```

Comment:
Let's create a more descriptive variable name. For now, it's not clear what should be stored here. Probably `joinedValues` will be better